packages/orchestration/src/exos/orchestrator.js

```diff
@@ -50,44 +51,89 @@ export const OrchestratorI = M.interface('Orchestrator', {
  *   orchestrationService: Remote<OrchestrationService>;
  *   storageNode: Remote<StorageNode>;
  *   timerService: Remote<TimerService>;
+ *   vowTools: VowTools;
  *   zcf: ZCF;
  * }} powers
  */
 export const prepareOrchestrator = (
```

Comment:
this is now a kit. `prepareOrchestratorKit`.

but no consumers need these other facets so the module should still export a `prepareOrchestrator` that returns a `makeOrchestrator` that returns just its `orchestrator` facet